listenbrainz/db/timescale.py

```diff
@@ -42,16 +42,25 @@ def run_sql_script_without_transaction(sql_file_path):
         connection = engine.connect()
         connection.connection.set_isolation_level(0)
         lines = sql.read().splitlines()
-        try:
-            for line in lines:
-                # TODO: Not a great way of removing comments. The alternative is to catch
-                # the exception sqlalchemy.exc.ProgrammingError "can't execute an empty query"
-                if line and not line.startswith("--"):
-                    connection.execute(line)
-        except sqlalchemy.exc.ProgrammingError as e:
-            print("Error: {}".format(e))
-            return False
-        finally:
-            connection.connection.set_isolation_level(1)
-            connection.close()
+        while True:
```

Comment:
```suggestion
        retries = 0
        while True:
```
We use `retries` in `except` clause but do not set it which can cause `NameError`.

Reply:
oops. bad copypasta.